sechub-doc/src/main/java/com/mercedesbenz/sechub/docgen/ExecutorConfigurationParameterDescriptionGenerator.java

```diff
@@ -0,0 +1,94 @@
+package com.mercedesbenz.sechub.docgen;
+
+import java.io.File;
+import java.util.List;
+
+import com.mercedesbenz.sechub.commons.pds.ExecutionPDSKey;
+import com.mercedesbenz.sechub.commons.pds.PDSKey;
+import com.mercedesbenz.sechub.commons.pds.PDSKeyProvider;
+import com.mercedesbenz.sechub.domain.scan.product.pds.PDSExecutorConfigSuppport;
+import com.mercedesbenz.sechub.pds.execution.PDSKeyToEnvConverter;
+
+public class ExecutorConfigurationParameterDescriptionGenerator {
+
+    private PDSKeyToEnvConverter keyToEnvConverter = new PDSKeyToEnvConverter();
+
+    public String generatePDSExecutorConfigurationParamters(File targetFile) {
+        StringBuilder sb = new StringBuilder();
+        addLine(".PDS executor configuration parameters", sb);
+        addLine("[%header,cols=3*]", sb);
+        addLine("|===", sb);
+        addLine("| Key", sb);
+        addLine("| Description", sb);
+        addLine("| Additional info", sb);
+
+        addLine("", sb);
+
+        List<PDSKeyProvider<? extends PDSKey>> list = PDSExecutorConfigSuppport.getUnmodifiableListOfParameterKeyProvidersSentToPDS();
+        for (PDSKeyProvider<? extends PDSKey> provider : list) {
+            PDSKey key = provider.getKey();
+            addLine("| " + key.getId(), sb);
+            addLine("|\n+++\n" + key.getDescription() + "\n+++\n", sb);
+            addLine("a| " + describe(provider), sb);
+            addLine("", sb);
+        }
+        sb.append("|===");
+
+        return sb.toString();
+    }
+
+    private String describe(PDSKeyProvider<? extends PDSKey> provider) {
+        StringBuilder sb = new StringBuilder();
+        PDSKey key = provider.getKey();
+        if (key != null) {
+            boolean isSentToPDS = key.isSentToPDS();
+            if (key instanceof ExecutionPDSKey) {
+                ExecutionPDSKey execKey = (ExecutionPDSKey) key;
+                if (execKey.isAvailableInsideScript()) {
+                    String env = keyToEnvConverter.convertKeyToEnv(key.getId());
+                    appendInfo("Available as ENV variable `" + env + "` inside launcher script.", sb);
+                }
+                if (!isSentToPDS) {
+                    /* fallback - the keyswhich are inside script are always et... */
```

Comment:
Instead of correcting the description, I drop the complete code (belonging to `isSentToPDS.)` Reason:  The if else construct does nothing. It is old code which is no longer used.